src/player.ts

```diff
@@ -320,13 +321,34 @@ export class Player extends NativeInstance<PlayerConfig> {
     return PlayerModule.isAirPlayAvailable(this.nativeId);
   };
 
+  /**
+   * @returns An array containing AudioTrack objects for all available audio tracks.
+   */
+  getAvailableAudioTracks = async (): Promise<AudioTrack[]> => {
+    return PlayerModule.getAvailableAudioTracks(this.nativeId);
+  };
+
+  /**
+   * Sets the audio track to the ID specified by trackIdentifier. A list can be retrieved by calling getAvailableAudioTracks.
+   */
+  setAudioTrack = async (trackIdentifier: string): Promise<void> => {
+    PlayerModule.setAudioTrack(this.nativeId, trackIdentifier);
+  };
+
   /**
    * @returns An array containing SubtitleTrack objects for all available subtitle tracks.
    */
   getAvailableSubtitles = async (): Promise<SubtitleTrack[]> => {
     return PlayerModule.getAvailableSubtitles(this.nativeId);
   };
 
+  /**
+   * Sets the subtitle track to the ID specified by trackIdentifier. A list can be retrieved by calling getAvailableSubtitles.
+   */
+  setSubtitleTrack = async (trackIdentifier: string): Promise<void> => {
```

Comment:
I think `trackIdentifier` should be optional. Our public API defines `setSubtitleTrack` with optional identifier parameter.